### PR TITLE
UPGRADE.md: added information how to make Elasticsearch work

### PR DESCRIPTION
**Pull request by simara-svatopluk**
| Q             | A
| ------------- | ---
|Description, reason for the PR| add info to upgrade.md how to make Elasticsearch run
|New feature| No 
|BC breaks| No 
|Fixes issues| ...
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
```
Comment (PetrHeinz) [inline UPGRADE.md]: might be too brief and misses a link to the docs

File: UPGRADE.md [hunk 2]
```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
+    - copy new `docker/conf/docker-compose.yml.dist` into your local `docker-compose.yml`
```
Comment (PetrHeinz) [inline UPGRADE.md]: it isn't as simple as copying the template, I would suggest generalizing it with "check changes in the `docker-compose.yml` template you used and replicate them", just as is written in upgrade from alpha1 to alpha2

File: UPGRADE.md (hunk 1 above)
Comment (PetrHeinz) [reply UPGRADE.md]: I would suggest: 
- for [product search via Elasticsearch](/docs/introduction/product-search-via-elasticsearch.md) to work properly, you'll have to:

`- for [product search via Elasticsearch](/docs/introduction/product-search-via-elasticsearch.md) to work properly, you'll have to:`

File: UPGRADE.md (hunk 2 above)
Comment (PetrHeinz) [reply UPGRADE.md]: I would suggest:
    - check changes in the `docker-compose.yml` template you used, a new container with Elasticsearch has to be added

`    - check changes in the ``docker-compose.yml`` template you used, a new container with Elasticsearch has to be added`

File: UPGRADE.md [hunk 3]
```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
+    - copy new `docker/conf/docker-compose.yml.dist` into your local `docker-compose.yml`
+    - run `docker-compose up -d`
+    - create Elasticsearch indexes by running `./phing elasticsearch-indexes-create`
```
Comment (vitek-rostislav) [inline UPGRADE.md]: please use `php phing` instead of `./phing` as the first one works on all platforms :slightly_smiling_face:

File: UPGRADE.md [hunk 4]
```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
+    - copy new `docker/conf/docker-compose.yml.dist` into your local `docker-compose.yml`
+    - run `docker-compose up -d`
```
Comment (vitek-rostislav) [inline UPGRADE.md]: I believe it must be `docker-compose up -d --force-recreate` as there are changes in the `docker-compose.yml`

File: UPGRADE.md [hunk 5]
```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
+    - copy new `docker/conf/docker-compose.yml.dist` into your local `docker-compose.yml`
+    - run `docker-compose up -d`
+    - create Elasticsearch indexes by running `./phing elasticsearch-indexes-create`
+    - export products into Elasticsearch by `./phing elasticsearch-products-export`
```
Comment (vitek-rostislav) [inline UPGRADE.md]: same here, `php phing`...

File: UPGRADE.md (hunk 2 above)
Comment (vitek-rostislav) [inline UPGRADE.md]: I would add information about platform, e.g. "copy new `docker/conf/docker-compose.yml.dist` (be aware that there are different `dist` files for each platform so you need to copy the proper one) into your local..."

File: UPGRADE.md (hunk 4 above)
Comment (PetrHeinz) [reply UPGRADE.md]: But the already running containers do not require recreating. I've tested it with just  `docker-compose -d` without any problems.

File: UPGRADE.md (hunk 3 above)
Comment (PetrHeinz) [reply UPGRADE.md]: Yeah, that would be better

File: UPGRADE.md (hunk 2 above)
Comment (PetrHeinz) [reply UPGRADE.md]: For the same reason I would change it to "check changes in the `docker-compose.yml` template you used, a new container with Elasticsearch has to be added"

**Pull request merged**